operate/qa/integration-tests/src/test/java/io/camunda/operate/schema/IndexRestartIT.java

```diff
@@ -0,0 +1,172 @@
+    } else {
+      operateProperties.getElasticsearch().setNumberOfReplicas(defaultNumberofReplicas);
+      operateProperties.getElasticsearch().setRefreshInterval(defaultRefreshInterval);
+    }
+
+    // update number of replicas for each index
+    schemaManager.checkAndUpdateIndices();
+
+    final Map<String, String> reindexSettings1 =
+        schemaManager.getIndexSettingsFor(idxName("index1"), NUMBERS_OF_REPLICA, REFRESH_INTERVAL);
+    assertThat(reindexSettings1)
+        .containsEntry(NUMBERS_OF_REPLICA, String.valueOf(defaultNumberofReplicas))
+        .containsEntry(
+            REFRESH_INTERVAL, DatabaseInfo.isElasticsearch() ? null : defaultRefreshInterval);
+
+    final Map<String, String> reindexSettings2 =
+        schemaManager.getIndexSettingsFor(idxName("index2"), NUMBERS_OF_REPLICA, REFRESH_INTERVAL);
+    assertThat(reindexSettings2)
+        .containsEntry(NUMBERS_OF_REPLICA, String.valueOf(defaultNumberofReplicas))
+        .containsEntry(
+            REFRESH_INTERVAL, DatabaseInfo.isElasticsearch() ? null : defaultRefreshInterval);
+
+    final Map<String, String> reindexSettings3 =
+        schemaManager.getIndexSettingsFor(idxName("index3"), NUMBERS_OF_REPLICA, REFRESH_INTERVAL);
+    assertThat(reindexSettings3)
+        .containsEntry(NUMBERS_OF_REPLICA, String.valueOf(defaultNumberofReplicas))
+        .containsEntry(
+            REFRESH_INTERVAL, DatabaseInfo.isElasticsearch() ? null : defaultRefreshInterval);
+
+    assertThat(schemaManager.getOrDefaultNumbersOfReplica(idxName("index1"), "2"))
+        .isEqualTo(Integer.toString(defaultNumberofReplicas));
+
+    assertThat(schemaManager.getOrDefaultRefreshInterval(idxName("index1"), "1s"))
+        .isEqualTo(DatabaseInfo.isElasticsearch() ? "1s" : defaultRefreshInterval);
+
+    assertThat(schemaManager.getOrDefaultNumbersOfReplica(idxName("index2"), "2"))
+        .isEqualTo(Integer.toString(defaultNumberofReplicas));
+
+    assertThat(schemaManager.getOrDefaultRefreshInterval(idxName("index2"), "1s"))
+        .isEqualTo(DatabaseInfo.isElasticsearch() ? "1s" : defaultRefreshInterval);
+
+    assertThat(schemaManager.getOrDefaultNumbersOfReplica(idxName("index3"), "2"))
+        .isEqualTo(Integer.toString(defaultNumberofReplicas));
+
+    assertThat(schemaManager.getOrDefaultRefreshInterval(idxName("index3"), "1s"))
+        .isEqualTo(DatabaseInfo.isElasticsearch() ? "1s" : defaultRefreshInterval);
+  }
+
+  private void createIndex(final String indexName, final List<Map<String, String>> documents)
+      throws Exception {
+    final Map<String, ?> mapping =
+        Map.of("properties", Map.of("test_name", Map.of("type", "keyword")));
+    searchRepository.createIndex(indexName, mapping);
+    for (final var document : documents) {
+      searchRepository.createOrUpdateDocument(indexName, UUID.randomUUID().toString(), document);
+    }
+  }
+
+  public class TestDescriptor extends AbstractIndexDescriptor implements Prio4Backup {
```

Comment:
There already is a 'TestIndex' class, please add to and utilize that if possible. Also, please don't create test utility classes inside test suite files, they should be kept separate.